packages/dcos-launch/extra/launch/onprem.py

```diff
@@ -0,0 +1,138 @@
+import copy
+import logging
+import subprocess
+
+import retrying
+import yaml
+
+import launch.aws
+import launch.util
+
+import test_util.aws
+import test_util.onprem
+import test_util.runner
+from test_util.helpers import Url
+
+log = logging.getLogger(__name__)
+
+STATE_FILE = 'LAST_COMPLETED_STAGE'
+
+
+@retrying.retry(wait_fixed=1000, stop_max_delay=120000)
+def check_ssh(ssher, host, port):
+    ssher.get_home_dir(host, port)
+
+
+class OnpremLauncher(launch.util.AbstractLauncher):
```

Comment:
This class has a `self.config` set in `__init__`, but several methods expect a separate `config` parameter. Is there a reason these methods (`create`, `post_state`, etc) are not using `self.config`?